packages/office-ui-fabric-react/src/components/Rating/examples/Rating.Basic.Example.tsx

```diff
@@ -44,7 +44,7 @@ export class RatingBasicExample extends React.Component<
           onChange={this._onLargeStarChange}
           onFocus={this._onFocus}
           onBlur={this._onBlur}
-          ariaLabelFormat={'{0} of {1} stars selected'}
+          ariaLabelFormat={'Select {0} of {1} stars'}
```

Comment:
any reason to change this text?

Reply:
@dzearing It was a request from @ecraig12345 above (resolved convo) in regards that what should be read out. The label should be geared towards the actual star the user is on and has the ability to select rather than indicating whether it's currently selected and its value.

Reply:
Alright, thanks :)